src/stream/src/executor/lookup/cache.rs

```diff
@@ -101,7 +101,7 @@ impl LookupEntryState {
         if self.inner.insert(value) {
             self.kv_heap_size.set(kv_heap_size);
         } else {
-            panic!("inserting a duplicated value");
+            crate::consistency::inconsistency_panic!("inserting a duplicated value");
```

Comment:
If `strict_consistency` is `off`, will we fail to do the maintenance on `kv_heap_size` here?

Reply:
If the insertion failed, the `kv_heap_size` should be unchanged.

Reply:
Oh, I see. `inner` is a set, so replacing the value should not affect the estimated size.

Reply:
But I still find this could be error-prone. 😕 For example, what if `inner` is a **map**? Developers have to manually handle the inconsistency-tolerated case.

Reply:
Yes...Seems this `LookupEntryState` should be `EstimatedHashSet` in `estimate_size` module and have nothing to do with the "data inconsistency" business

Reply:
Let me revert the changes in `Lookup` in this PR, will refactor the `LookupEntryState` in future PR